base/system/logonui/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+
+add_executable(logonui logonui.c logonui.rc NT5design.c)
```

C files first, then the resources.
```suggestion
add_executable(logonui logonui.c NT5design.c logonui.rc)
```